test/sequential/bootstrap.checkActivatePool.test.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable jest/no-conditional-expect */
+/*
+ *
+ * @group bootstrap
+ * @group sequential
+ */
+import { getApi, initApi, getMangataInstance } from "../../utils/api";
+import {
+  scheduleBootstrap,
+  provisionBootstrap,
+  finalizeBootstrap,
+  getLiquidityAssetId,
+  claimAndActivateBootstrap,
+} from "../../utils/tx";
+import { EventResult, ExtrinsicResult } from "../../utils/eventListeners";
+import { Keyring } from "@polkadot/api";
+import { User } from "../../utils/User";
+import {
+  getEnvironmentRequiredVars,
+  waitForBootstrapStatus,
+} from "../../utils/utils";
+import {
+  getEventResultFromMangataTx,
+  sudoIssueAsset,
+} from "../../utils/txHandler";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+import { BN, toBN } from "@mangata-finance/sdk";
+import { Assets } from "../../utils/Assets";
+import { Sudo } from "../../utils/sudo";
+import { setupApi, setupUsers } from "../../utils/setup";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.spyOn(console, "error").mockImplementation(jest.fn());
+jest.setTimeout(3500000);
+process.env.NODE_ENV = "test";
+
+let testUser1: User;
+let sudo: User;
+let keyring: Keyring;
+let bootstrapPhase: any;
+let bootstrapCurrency: any;
+let bootstrapPool: any;
+let eventResponse: EventResult;
+
+const { sudo: sudoUserName } = getEnvironmentRequiredVars();
+const waitingPeriod = 15;
+const bootstrapPeriod = 30;
+const whitelistPeriod = 10;
+const bootstrapAmount = new BN(10000000000);
+
+async function checkingPool() {
```

Comment:
rename this function with something more verbose, and avoid the use of global variables. for example: `validatePollCreationFails(tokenA,TokenB)`